docs/stylesheets/extra.css

```
@@ -1,24 +1,25 @@
   --ok-fast: 140ms var(--ok-ease);
   --ok-med: 220ms var(--ok-ease);
   --ok-slow: 520ms var(--ok-ease);
-  --md-sidebar-width: 12.5rem;
+  --ok-sidebar-base: min(12.5rem, 20vw);
+  --md-sidebar-width: var(--ok-sidebar-base);
+  --ok-content-max: min(56.25rem, 90vw);
+  --md-header-height: clamp(80px, 15vh, 140px);
 }
 
-/* Theme-aware surfaces */
+/* THEME SYSTEM: Defines distinct color palettes for Light (default) and Dark (slate) modes*/
```

Stylelint (17.7.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)